index.bs

```diff
@@ -918,14 +918,19 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 : <dfn>Client-side-resident Public Key Credential Source</dfn>
 : <dfn>Resident Credential</dfn>
-:: A [=Client-side-resident Public Key Credential Source=], or [=Resident Credential=] for short, is a [=public key credential
-    source=] whose [=credential private key=] is stored in the [=authenticator=], [=client=] or [=client device=]. Such
-    [=client-side=] storage requires a [=resident credential capable=] [=authenticator=] and has the property that the [=authenticator=]
-    is able to select the [=credential private key=] given
-    only an [=RP ID=], possibly with user assistance (e.g., by providing the user a pick list of [=public key credential|credentials=] [=scoped=] to the [=RP ID=]).
-    By definition, the [=credential private key=] is always exclusively controlled by the [=authenticator=]. In the case of a
-    [=resident credential=], the [=authenticator=] might offload storage of wrapped key material to the
-    [=client device=], but the [=client device=] is not expected to offload the key storage to remote entities (e.g., [=[WRP]=] Server).
+:: A [=Client-side-resident Public Key Credential Source=], or [=Resident Credential=] for short,
+    is a [=public key credential source=] that can be used in an [=authentication ceremony=]
+    without providing the [=authenticator=] with a [=credential ID=], i.e.,
+    in an [=authentication ceremony=] with an [=list/empty=] {{PublicKeyCredentialRequestOptions/allowCredentials}} argument.
+
+    As a consequence, a [=resident credential capable=] [=authenticator=] can generate an [=assertion signature=]
+    for a [=resident credential=] given only an [=RP ID=],
+    which in turn means that the corresponding [=credential private key=]
+    is stored in the [=authenticator=], [=client=] or [=client device=].
+    This is in contrast to a [=server-side-resident public key credential source=],
+    which requires that the [=authenticator=] is given both the [=RP ID=] and the [=credential ID=]
+    but does not require [=client-side=] storage of the [=credential private key=].
```

Comment:
should "[=credential private key=]" here be "[=public key credential source=]" ?